agent/pkg/models/models.go

```diff
@@ -105,33 +97,8 @@ type ExtendedCreator struct {
 	Source *string `json:"_source"`
 }
 
-type FullEntryWithPolicy struct {
-	RulesMatched []rules.RulesMatched `json:"rulesMatched,omitempty"`
-	Entry        har.Entry            `json:"entry"`
-	Service      string               `json:"service"`
+func RunValidationRulesState(harEntry har.Entry, service string) (tapApi.ApplicableRules, []rules.RulesMatched) {
+	numberOfRules, resultPolicyToSend := rules.MatchRequestPolicy(harEntry, service)
```

Comment:
here there is a weird thing.. 
the numnerOfRules is a parameter defined when it is returned from MatchRequestPolicy function and then reassigned in the seconds function? (and I see the second parameter to the PassedValidationRules function is redundant (because it is not used inside)..
again - maybe I miss something but I think there is small fix needed here...

Reply:
Yes you are right, this is from a really old implementation but that it was present until now, removed